Source/Extensions/Blazorise.DataGrid/Models/DataGridSortColumn.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Blazorise.DataGrid;
+
+/// <summary>
+/// A column sort definition
+/// </summary>
+public class DataGridSortColumn
```

Comment:
What is the difference between `DataGridSortColumnInfo` and `DataGridSortColumn`?

Reply:
Again I believe I've had the same question & @danm-de has clarified it before, I'll re-check the reviews above. It does seem like a bit of uneeded duplication though on first look.

Reply:
From what I can tell, the `DataGridSortColumn` is used as an input for `ApplySorting`, as such it only has the necessary information for this operation.
The `DataGridSortColumnInfo` holds concrete useful information about the column sorting attributes.

This seems fine to me. However I have doubts about the `DataGridSortColumn` naming since it's more like an input dto and not really an actual `Column` like it would seem. I think we've gonne through renaming before on this though haha

Reply:
So do you feel we should make up a convention for this type of input based arguments? Just that the name is similar to `DataGridColumn`, `DataGridDateColumn`, `DataGridCommandColumn`, etc which hold a different meaning, they are actual child `DataGridColumns` types of `DataGrid`

`DataGridSortColumnArgs`? `DataGridSortColumnDefinition`? `DataGridSortColumnInput`?

Reply:
In the past, I have created an `*Info` class. For example `DataGridColumnInfo` for `DataGridColumn` have simple data to be used by the grid without having full reference to the DataGridColumn. So the _Info_ is an OK suffix to have. The `Definition` is also used in some places.

Reply:
I believe we are using the `Info` as actual `*.Info` provided by events etc... That's how the `DataGridSortColumnInfo` was being used also.
![image](https://github.com/Megabit/Blazorise/assets/22283161/4ff69e16-d975-43a9-8b81-3304f765ecb4)

in this case you're providing a method with an argument / input. So using Info here could be slightly different, but I guess we can do that. We're removing `SortingChanged` and can repurpose `DataGridSortColumnInfo`